.github/workflows/test.yml

```
@@ -0,0 +1,90 @@
+name: Timber starter theme tests
+
+on:
+  push:
+    branches:
+      - '*'
```

```suggestion
  push:
    branches:
      - '1.x'
      - '2.x'
```

It will prevent double runs.

---

Alright, let’s try it out! Thanks for looking into the details 😎.

---

22 👉🏻 11